Automattic/jetpack
# VideoPress : remove "download" from share options

Related to 988-gh-Automattic/greenhouse

#### Changes proposed in this Pull Request:

Change "share" option text for video in the media library to remove "download" in it.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a jetpack site with VideoPress enabled (/wp-admin/admin.php?page=jetpack#/dashboard "VideoPress" should be toggled on)
* Go to your media library /wp-admin/upload.php
* If you don't already have a VideoPress video, upload a video now
* ✅ Click on the video, the following text should appear next to "Share"

![Capture d’écran 2021-10-12 à 16 11 55](https://user-images.githubusercontent.com/1420594/136972555-cd1e7b65-4ad2-4d6b-9ccd-5ddf1c124bf4.png)
